css-ui-4/Overview.bs

```diff
@@ -2199,6 +2203,42 @@ that are typical of the element whose appearance it acquires.
 	is also not changed by the 'appearance' property.
 </div>
 
+<h3 id="input-security">Obscuring sensitive input: the 'input-security' property</h3>
+
+<pre class=propdef>
+Name: input-security
+Value: auto | none
+Initial: ''input-security/auto''
+Applies to: elements that accept sensitive text input, such as <{input/type/password|&lt;input type=password&gt;}>
```

Comment:
This makes me vaguely uncomfortable. Is it indeed intended that this only works on password inputs, not on text inputs? If so, we should probably be clearer that this is host-language defined. Probably just say "Applies to: [=sensitive-text inputs=]" and define the term in the text as being host-language defined, and either specifying that in HTML only password inputs are sensitive-text inputs or linking to the spot you'll add in HTML that says the same.

Reply:
> This makes me vaguely uncomfortable. Is it indeed intended that this only works on password inputs, not on text inputs?

That was the resolution. I'm not super enthused about this either—as I said in the original issue, it's sometimes desirable to obscure the text of other inputs, such as `<input type=tel>`. But I figure that's a discussion we can have in a followup issue.

> If so, we should probably be clearer that this is host-language defined. Probably just say "Applies to: [=sensitive-text inputs=]" and define the term in the text as being host-language defined, and either specifying that in HTML only password inputs are sensitive-text inputs or linking to the spot you'll add in HTML that says the same.

Okay, I'll do something like this.

Reply:
I've given this a go in 3f5738a. Please let me know what you think, @tabatkins!